src/app/containers/utils/multicall.ts

```diff
@@ -0,0 +1,146 @@
+import ERC_20 from "abi/erc20.json";
+import LP_TOKEN_UNGUARDED from "abi/lpTokenUnguarded.json";
+import MASTERCHEF from "abi/masterchef.json";
+import { AXIAL_MASTERCHEF_CONTRACT_ADDRESS } from "app/constants";
+import { ChainId } from "app/constants";
+import { Multicall } from "ethereum-multicall";
+import { ethers } from "ethers";
+
+//we need to disable all unsafe protections for any types
+//because all returns from this lib are generic
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Lots of fighting with the linter, should we even consider using it we need to constantly be fighting it?, otherwise please lets try to adhere to it as much as possible.

Reply:
all of these ignores are coming from old Axial logic
every ignore that is written by me has a reason on top of it

Reply:
Lets always fix and adjust the code when porting over the old structure, we cant continue to carry out as it is. Otherwise will endup with the same old code.